src/gui/Connected.qml

```diff
@@ -57,29 +58,29 @@ Item {
     property string meterYellow: "#F5BF4F"
     property string meterRed: "#F21B1B"
 
-    property bool isUsingRtAudio: virtualstudio.audioBackend == "RtAudio"
+    property bool isUsingRtAudio: audio.audioBackend == "RtAudio"
 
     function getCurrentInputDeviceIndex () {
-        if (virtualstudio.inputDevice === "") {
-            return virtualstudio.inputComboModel.findIndex(elem => elem.type === "element");
+        if (audio.inputDevice === "") {
+            return audio.inputComboModel.findIndex(elem => elem.type === "element");
         }
 
-        let idx = virtualstudio.inputComboModel.findIndex(elem => elem.type === "element" && elem.text === virtualstudio.inputDevice);
+        let idx = audio.inputComboModel.findIndex(elem => elem.type === "element" && elem.text === audio.inputDevice);
         if (idx < 0) {
-            idx = virtualstudio.inputComboModel.findIndex(elem => elem.type === "element");
+            idx = audio.inputComboModel.findIndex(elem => elem.type === "element");
         }
 
         return idx;
     }
 
     function getCurrentOutputDeviceIndex() {
```

Comment:
I think we can actually delete `getCurrentOutputDeviceIndex()` and `getCurrentOutputDeviceIndex()` in this file

Reply:
removed